Haskell/MaîtreDTests.hs

```diff
@@ -0,0 +1,61 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+module MaîtreDTests
+    ( tryAcceptBehavesCorrectlyWhenItCanAccept
+    , tryAcceptBehavesCorrectlyWhenItCanNotAccept
+    ) where
+
+import           Control.Monad      (liftM2)
+import           Data.Maybe         (isNothing)
+import           Data.Time          (LocalTime (..), ZonedTime (..),
+                                     defaultTimeLocale, iso8601DateFormat,
+                                     midnight, parseTimeOrError, utc)
```

Comment:
Aren't `defaultTimeLocale`, `iso8601DateFormat`, and `parseTimeOrError` now redundant?